main.js

```diff
@@ -387,6 +387,8 @@ async function createWindow() {
   // Note: We do most of our shutdown logic here because all windows are closed by
   //   Electron before the app quits.
   mainWindow.on('close', async e => {
+    e.preventDefault();
```

Comment:
Looks like this isn't what we want, based on the comments on line 396. Unfortunately window management on the various platforms and modes is a little complex. So please try to make the most minimal fix which will work, and be sure to test it on Windows, Linux and macOS.

Reply:
Ah, I missed that comment. I'll revert that change. I can test on my windows and linux tonight.